Mathlib/Order/Closure.lean

```diff
@@ -103,9 +125,14 @@ variable {α} [PartialOrder α] (c : ClosureOperator α)
 
 @[ext]
 theorem ext : ∀ c₁ c₂ : ClosureOperator α, (c₁ : α → α) = (c₂ : α → α) → c₁ = c₂ :=
-  DFunLike.coe_injective
+  @DFunLike.ext' _ _ _ _
 #align closure_operator.ext ClosureOperator.ext
 
+-- this should probably be the defautlt?
```

Comment:
```suggestion
```
It looks to me like you're right. If so, now is the time to enact this! Can you try it (and also have only one lemma carrying `@[ext]`).